patroni/ha.py

```diff
@@ -1379,7 +1379,10 @@ def is_failover_possible(self, *, cluster_lsn: int = 0, exclude_failover_candida
                 quorum_votes += 1
 
         # In case of quorum replication we need to make sure that there is enough healthy synchronous replicas!
-        return quorum_votes >= (self.cluster.sync.quorum if self.quorum_commit_mode_is_active() else 0)
+        # However, when failover candidate is set, we can ignore quorum requirements.
+        quorum = self.cluster.sync.quorum if self.quorum_commit_mode_is_active() and\
+            not (self.cluster.failover and self.cluster.failover.candidate and not exclude_failover_candidate) else 0
+        return quorum_votes >= quorum
```

Comment:
looks like we are a bit uninformative about the reason of the not possible failover/switchover when quorum requirement is not satisfied. can we maybe add some info like for the checks above?